llvm/test/CodeGen/AMDGPU/sdwa-peephole-vcnd_mask-2.mir

```diff
@@ -0,0 +1,43 @@
+# NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py UTC_ARGS: --version 5
+# RUN: llc %s -mtriple=amdgcn -mcpu=gfx1030 -run-pass=si-peephole-sdwa -o - | FileCheck %s
+
+# For conversion of_CNDMASK_B32_e64 to SDWA, the destination of V_CMP_O_F16_e64 must be
+# changed to vcc_lo first. This would introduce a vcc_hi use that requires special
+# handling in si-peephole-sdwa.
+
+---
```

Comment:
These different tests should go in the same test file

Reply:
They target incompatible ISAs, i.e. putting them in the same file crashes because of invalid register sizes.

Reply:
you need to split wave32 vs. wave64, not by testcase

Reply:
I have tests for wave32 and wave64 in separate files now.